roadmap.md

```diff
@@ -93,7 +93,7 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 ### Cookies
 
-This scenario loads a web page and gets and sets its cookies. In spec terms, this involves:
+This scenario loads a web page, reads the cookies set by the server and the page, updates and deletes some of the cookies, and triggers an action to verify that modified cookies are processed correctly. In spec terms, this involves:
```

Comment:
This doesn't cover cookies for pages that aren't loaded. My understanding is that the need to load a page before interacting with its cookies is considered a significant limitation for WebDriver. It's also the source of much of the complexity here (since having a page loaded in a specific context answers a lot of the questions around keying, given that a particular browsing context will have a particular storage key).

Reply:
Is this addressed by the edit that happened after this comment was posted?